examples/tailwindcss/package.json

```diff
@@ -1,35 +1,38 @@
 {
-  "private": true,
   "name": "remix-example-tailwindcss",
+  "private": true,
   "description": "",
   "license": "",
+  "sideEffects": false,
   "scripts": {
-    "build": "npm run build:css && remix build",
-    "build:css": "tailwindcss -o ./app/tailwind.css",
-    "dev": "concurrently \"npm run dev:css\" \"remix dev\"",
-    "dev:css": "tailwindcss -o ./app/tailwind.css --watch",
+    "build": "run-p build:*",
+    "build:css": "npm run generate:css -- --minify",
+    "build:remix": "cross-env NODE_ENV=production remix build",
+    "dev": "run-p dev:*",
+    "dev:css": "npm run generate:css -- --watch",
+    "dev:remix": "cross-env NODE_ENV=development remix dev",
+    "generate:css": "npx tailwindcss -o ./app/tailwind.css",
     "postinstall": "remix setup node",
-    "start": "remix-serve build"
+    "start": "cross-env NODE_ENV=production remix-serve build"
   },
   "dependencies": {
     "@remix-run/react": "1.2.2",
     "@remix-run/serve": "1.2.2",
+    "cross-env": "^7.0.3",
```

Comment:
maybe we could move it to devDependencies

Reply:
Just copied over the base template, but should indeed be in `devDependencies` 👍

Fixed the template in #2157

Reply:
`cross-env` is used in the start script, it is in fact a runtime dependency and therefore belongs in `dependencies` not `devDependencies`.

Reply:
@jacob-ebey Shouldn't it be a dependency of `remix` itself then? 🤔